docs/.htmltest.yml

```diff
@@ -0,0 +1,3 @@
+DirectoryPath: website/public
+IgnoreDirectoryMissingTrailingSlash: true
+CheckExternal: false
```

Comment:
We should probably set `IgnoreAltMissing` or see if we can filter the results more.. as-is the output is kind of crazy with the versioned content.. its like `2937 errors in 267 documents` where almost all of them are missing alt tags in old content (same ones are repeated for each version).